### ISSUE TITLE
v1.9.1 onwards depends on ansible.windows which does not exist

### ISSUE THREAD

**Issue by anshulbehl** [sensu/sensu-go-ansible#275]
**Bug description**
A clear and concise description of what the issue is.
There is a dependency on ansible.windows which does not exist on Automation Hub, hence its failing the building of execution environment
Also any customer who will try to install the collections from Automation Hub will also fail because of the unavailability of the collection, please fix this asap

cc: @tadeboro

**Comment by tadeboro** [comment]
Hi @anshulbehl.

We added the `ansible.windows` dependency when we were preparing the Sensu Go Ansible Collection for execution environments. The reasoning behind our decision was that without Windows collection, EE will fail to execute valid playbooks if users are trying to manage Windows hosts. What I failed to notice at the time was the fact that the `ansible.windows` collection is not certified (I naively assumed anything in the `ansible` namespace is certified) and only tested EE build using Ansible Galaxy as the source of collections.

Before the introduction of execution environments, this was not an issue because all of the relevant Windows content was part of Ansible 2.9 and our roles are written with Ansible 2.9 users in mind (we do not use FQCNs in our roles for this reason).

We have a PR ready that removes the hard dependency on the Windows Ansible Collection, but there is a really nasty side-effect of that PR: EEs containing the Sensu Go Ansible Collection just lost the capability to install Sensu Go agents on Windows hosts. So community users actually have access to more functionality than paying customers that rely solely on certified content.
